File: LeetCode/Java/709-Unknown-转换成小写字母.java
/*
 * @lc app=leetcode.cn id=709 lang=java
 *
 * [709] 转换成小写字母
 */

// @lc code=start
class Solution {
    public String toLowerCase(String str) {
        var builder = new StringBuilder();
        for (char c : str.toCharArray()) {
            if (c >= 'A' && c <= 'Z')
                builder.append((char) (c + 'a' - 'A'));
            else
                builder.append(c);
        }
        return builder.toString();
    }
}
// @lc code=end

File: LeetCode/C/709-Unknown-转换成小写字母.cpp
/*
 * @lc app=leetcode.cn id=709 lang=cpp
 *
 * [709] 转换成小写字母
 */
#include "main.h"

// @lc code=start
class Solution {
  public:
    string toLowerCase(string str) {
        for (int i = 0; i < str.size(); i++)
            if (str[i] >= 'A' && str[i] <= 'Z')
                str[i] += 'a' - 'A';
        return str;
    }
};
// @lc code=end
